src/ai/backend/manager/api/session.py

```diff
@@ -407,18 +407,16 @@ async def _create(request: web.Request, params: Any) -> web.Response:
         if len(alias_folders) != len(set(alias_folders)):
             raise InvalidAPIParameters('Duplicate alias folder name exists.')
 
-        p: str
-        for p in alias_folders:
-            if p is None:
+        alias_name: str
+        for alias_name in alias_folders:
+            if alias_name.startswith("/home/work/"):
+                alias_name = alias_name.replace('/home/work/', '')
+            if alias_name is None:
                 continue
```

Comment:
```suggestion
            if alias_name is None:
                continue
            if alias_name.startswith("/home/work/"):
                alias_name = alias_name.replace('/home/work/', '')
```
The `None` check should be placed before calling any method.